### PR TITLE
Added solution for Dataminr Pulse

### PR DESCRIPTION
**Pull request by jayeshprajapaticrest**
Required items, please complete
   
   Change(s):
   - Added Dataminr Pulse solution for certification

   Reason for Change(s):
   - New Product

   Version Updated:
   - Initial Version

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml
Comment (oshezaf) [inline Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml:1]: This is not a valid use of an alert rule. Creating an incident for each event to generate an automation side effect will strain the system. Also, while you can "auto-close" the incident, the side effects on anything from machine learning to SOC optimization will be significant. You need to find a different method to apply your enrichment.

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml
Comment (oshezaf) [inline Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml:1]: Your events are alerts and not audit events, and therefore the parser is irrelevant. Please remove it.

**Review by oshezaf — CHANGES_REQUESTED** [review]
Tagging @devikamehra , @kavishbakshi1, and @anki-narravula to the comments in my review as they are fundamental.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest oshezaf have requested changes

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest oshezaf have requested changes

@v-prasadboke We have noted comment suggested by @oshezaf and we are working on it.

File: Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml
Comment (jayeshprajapaticrest) [reply Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml:1]: @oshezaf 
Is it fine if we remove the Analytic rule which is creating the Incident, but keep the playbook so that any prospective user can trigger this playbook based on his preferences which can be dependent on any other custom log table data?

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml
Comment (jayeshprajapaticrest) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml:1]: @oshezaf 
We did not find any direct schema with which we can map our alert data, So we picked up Audit events which had maximum number of matching fields with our data. If you think we can map our alert data with any other ASIM schema then please suggest.

Comment (oshezaf) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml:1]: While we do consider one, at this point in time there is no schema for alerts and therefore no ASIM parser is called for. You create an alert from your event by creating an analytic rules that maps entities to the (non-ASIM) alert schema. 

In practice, just remove the ASIM parser. You can have a non-ASIM parser that uses the field names, but not as part of the ASIM distribution but rather as part of the solution.

File: Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml
Comment (oshezaf) [reply Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml:1]: Yes. Of course.

Comment (jayeshprajapaticrest) [reply Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml:1]: @oshezaf Thanks We will do it.

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml
Comment (jayeshprajapaticrest) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml:1]: @oshezaf OK Thanks

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest any updates on the above

File: Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml
Comment (jayeshprajapaticrest) [reply Solutions/Dataminr Pulse/Analytic Rules/DataminrAlertEnrichmentASimDns.yaml]: @oshezaf 
As discussed above we have removed the analytics rule which rans the workflow for dataminr alert enrichment.

Cc: @v-prasadboke

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml
Comment (jayeshprajapaticrest) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml:1]: @oshezaf @v-prasadboke 
Regarding the ASIM parser we are working on to find the best possible schema to match our dataminr data.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest Please remove the branch conflicts as well

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest Please remove the branch conflicts as well

@v-prasadboke Sure we will do it. Thanks

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest any updates on the above

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest any updates on the above

@v-prasadboke I have resolved the conflicts. Please confirm and let me know if anything else changes are required.

**Comment by v-prasadboke** [comment]
Hello @oshezaf author has requested for review

**Comment by v-prasadboke** [comment]
Hello @oshezaf any updates on the above

File: Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/azuredeploy.json
Comment (v-prasadboke) [inline Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/azuredeploy.json:80]: Hello @jayeshprajapaticrest value set for variable "Keyvault-audience" is "vault.azure.net". 
That is where the arm ttk is failing.
Does that variable should refer to the parameter "BaseURL" ???

**Comment by v-prasadboke** [comment]
The api version of Microsoft.Resources/templateSpecs/versions is 2021-05-01 please replace it with 2022-02-01.
Please correct the api version and repackage the solution again.

**Comment by jayeshprajapaticrest** [comment]
> The api version of Microsoft.Resources/templateSpecs/versions is 2021-05-01 please replace it with 2022-02-01. Please correct the api version and repackage the solution again.

@v-prasadboke I have updated API version and committed the updated solution.

File: Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/azuredeploy.json
Comment (jayeshprajapaticrest) [reply Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/azuredeploy.json:80]: @v-prasadboke No. It does not refer to the parameter BaseUrl as this Url is for APIs used in the playbook.

**Comment by jayeshprajapaticrest** [comment]
> > The api version of Microsoft.Resources/templateSpecs/versions is 2021-05-01 please replace it with 2022-02-01. Please correct the api version and repackage the solution again.
> 
> @v-prasadboke I have updated API version and committed the updated solution.

As we are done with the API version change is there any update on this PR for further process?

**Comment by jayeshprajapaticrest** [comment]
@oshezaf As there is no specific schema available to map our dataminr alerts data. We have scrapped that use case of ASIM mapping and remove the related files from the PR.

**Comment by jayeshprajapaticrest** [comment]
@v-atulyadav @devikamehra @v-prasadboke @manishkumar1991 @oshezaf @rahul0216 @manishkumar1991 
Is there any updates on this PR?

File: Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/README.md
Comment (manishkumar1991) [inline Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/README.md:23]: spell check : paramteres

**Comment by manishkumar1991** [comment]
@jayeshprajapaticrest : Kindly share the playbook working screenshots

**Comment by jayeshprajapaticrest** [comment]
@Team
While resolving conflicts don't know PR gets automatically closed.

File: Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/README.md
Comment (jayeshprajapaticrest) [reply Solutions/Dataminr Pulse/Playbooks/DataminrPulseAlertEnrichment/README.md]: Done

**Comment by jayeshprajapaticrest** [comment]
> @jayeshprajapaticrest : Kindly share the playbook working screenshots
@rahul0216 PFA for the same.

PlaybookDataminr

File: Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml
Comment (jayeshprajapaticrest) [reply Parsers/ASimAuditEvent/Parsers/ASimAuditEventDataminrPulse.yaml]: @oshezaf We have removed the ASIM parse because of not finding the suitable schema to map Dataminr Alerts.

**Comment by manishkumar1991** [comment]
Playbook looks good.

**Review by manishkumar1991 — APPROVED** [review]
Playbook approved

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest there is still one arm-ttk failure for the PR, please resolve it

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest there is still one arm-ttk failure for the PR, please resolve it

@v-prasadboke 
Working on it. Still getting the hard coded URL related validation error even if I used variable to get it from customer as input with default values.

**Comment by v-prasadboke** [comment]
Ok @jayeshprajapaticrest will examine the validation failures, You can expect an update till 18 Jul

**Comment by jayeshprajapaticrest** [comment]
> Ok @jayeshprajapaticrest will examine the validation failures, You can expect an update till 18 Jul

OK. Thanks.
Then should I commit updated solution created using V3 tool?

File: Workbooks/WorkbooksMetadata.json
Comment (v-prasadboke) [inline Workbooks/WorkbooksMetadata.json:1]: Hello @jayeshprajapaticrest please remove workbook metadata from this file and include it in tools/create-azure-solution/v2/workbookmetadata/workbookmetadata.
And continue with V3

Comment (jayeshprajapaticrest) [reply Workbooks/WorkbooksMetadata.json:1]: Done as suggested.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, arm ttk is failing for hardcoded uri. As discussed in the call ill have this issue investiagted and Update you soon.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke Any updates on this PR?

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest can we have a quick call for this PR.

If it takes less than 15 min than sure we can connect now. Please share the link.

**Comment by v-prasadboke** [comment]
Shared the link, you can join.

**Comment by jayeshprajapaticrest** [comment]
> Shared the link, you can join.

I am not able to find it. Can you share here or mailed me?

**Comment by v-prasadboke** [comment]
<div style="margin-top:24px; margin-bottom:20px"><span style="font-size:24px; color:#252424">Microsoft Teams meeting</span>
</div>
<div style="margin-bottom:20px">
<div style="margin-top:0px; margin-bottom:0px; font-weight:bold"><span style="font-size:14px; color:#252424">Join on your computer, mobile app or room device</span>
</div>
<a style="font-size:14px; font-family:'Segoe UI Semibold','Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif; text-decoration:underline; color:#6264a7" class="me-email-headline" href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_YTJkZGQwY2ItZDM0NC00NDcyLThhNGItZGZjZmJkNWZlZGMy%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%2228263f0a-981b-4a56-a4c9-1e91e9ea44fb%22%7d">Click
 here to join the meeting</a> </div>
<div style="margin-bottom:20px; margin-top:20px">
<div style="margin-bottom:4px"><span style="font-size:14px; color:#252424" data-tid="meeting-code">Meeting ID:
<span style="font-size:16px; color:#252424">234 142 451 259</span> </span><br>
<span style="font-size:14px; color:#252424">Passcode: </span><span style="font-size:16px; color:#252424">RcEq4U
</span>
<div style="font-size:14px"><a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://www.microsoft.com/en-us/microsoft-teams/download-app">Download
 Teams</a> | <a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://www.microsoft.com/microsoft-teams/join-a-meeting">
Join on the web</a></div>
</div>
</div>
<div style="margin-bottom:4px">
<div style="margin-bottom:4px"><span style="font-size:14px; color:#252424; font-weight:bold">Or call in (audio only)</span>
</div>
<div style="margin-bottom:4px"><a style="font-size:14px; text-decoration:underline; color:#6264a7" href="tel:+13238494874,,331331961#">[+1 323-849-4874,,331331961#](tel:+13238494874,,331331961#)</a>
<span style="font-size:14px; color:#252424">&nbsp; United States, Los Angeles </span></div>
</div>
<span style="font-size:14px; color:#252424">Phone Conference ID: </span><span style="font-size:16px; color:#252424">331 331 961#
</span>
<div style="margin-bottom:20px"><a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://dialin.teams.microsoft.com/8551f4c1-bea3-441a-8738-69aa517a91c5?id=331331961">Find
 a local number</a> | <a style="font-size:14px; text-decoration:underline; color:#6264a7" class="me-email-link" href="https://dialin.teams.microsoft.com/usp/pstnconferencing">
Reset PIN</a> </div>
<div style="margin-bottom:24px; margin-top:20px"><a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://aka.ms/JoinTeamsMeeting">[Learn More](https://aka.ms/JoinTeamsMeeting)</a>
 | <a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://teams.microsoft.com/meetingOptions/?organizerId=28263f0a-981b-4a56-a4c9-1e91e9ea44fb&amp;tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&amp;threadId=19_meeting_YTJkZGQwY2ItZDM0NC00NDcyLThhNGItZGZjZmJkNWZlZGMy@thread.v2&amp;messageId=0&amp;language=en-US">
Meeting options</a> </div>

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, as discussed in the call you can go with the requested changes.
https://github.com/Azure/Azure-Sentinel/blob/master/Playbooks/Update-NamedLocations-TOR/azuredeploy.json 

You can refer this Playbook for more clarification.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest any updates on the requested changes.

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest any updates on the requested changes.

Just created the solution locally with the updated changes of key vault url as discussed in the call. But still getting the hard coded url related validation.
Will be updating it in this PR in an hour.

**Comment by v-prasadboke** [comment]
Ok sure, Thanks for the updates @jayeshprajapaticrest.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
Validation gets succeeded. Can you please proceed with the PR merging process?

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Thank you for the updates but still an arm ttk validation is failing for the same

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest, Thank you for the updates but still an arm ttk validation is failing for the same

Yes today again we got the error. I did the changes as suggested. But still getting this error.
Can you please help us out so that PR gets merged as early as possible?

**Comment by v-prasadboke** [comment]
Found out what was the issue
can we go with a quick call.

**Comment by v-prasadboke** [comment]
https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZmE2ZTIzYmUtZTBkMi00ZjNkLTg1ZDgtZjM1N2VkZWUzYTFi%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%2228263f0a-981b-4a56-a4c9-1e91e9ea44fb%22%7d

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, we are still discussing this issue internally. You can expect an update till 02 Aug 2023

**Comment by v-prasadboke** [comment]
Hello there, @jayeshprajapaticrest. This is still being worked on. Sorry for the trouble and delay in getting this PR Merged. 
I'm looking for a solution to the validation check that is failing for hardcoded URIs.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, as discussed in the call we are still onto this and update you soon about the same.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, We are still discussing this check failure for hard coded uri. Will update you before 18 Aug 2023.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Sorry for the inconvenience. But we are facing some technical issue while working on the logic app design. 
This is my assurance I'll update provide you some update and fix the issue of this playbook.
Thank you and Once again I apologize for the delay.

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest, Sorry for the inconvenience. But we are facing some technical issue while working on the logic app design. This is my assurance I'll update provide you some update and fix the issue of this playbook. Thank you and Once again I apologize for the delay.

@v-prasadboke 
Understood. Thanks for the reply.
Please try to do as fast as you can as we need this integration publicly available.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
Can we reschedule it to early at around 12:30 if possible?

**Comment by jayeshprajapaticrest** [comment]
> @v-prasadboke Can we reschedule it to early at around 12:30 if possible? As I have some meetings between 1 to 4 PM today.

@v-prasadboke 
My meeting gets cancelled. So we can connect at 3:00 PM. If possible from your end please share the link.
Thanks.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, If you are available for tomorrow 24 August, 2023 can we schedule a call?

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest, If you are available for tomorrow 24 August, 2023 can we schedule a call?

@v-prasadboke Sure we can connect.

**Comment by v-prasadboke** [comment]
Thank you for the reply @jayeshprajapaticrest, Will share you the invite within some time.

**Comment by jayeshprajapaticrest** [comment]
> Thank you for the reply @jayeshprajapaticrest, Will share you the invite within some time.

@v-prasadboke 
Are we going to connect as I didn't have the invite?

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, I've sent you a mail with attached .json file of the playbook. Please check it out and let me know about the same.
Thanks, Prasad.

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest, I've sent you a mail with attached .json file of the playbook. Please check it out and let me know about the same. Thanks, Prasad.

@v-prasadboke 
Thanks for the update.
Will check and updates you here as soon as possible.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Any updates on the above requested changes.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Could you please share some updates on the above forementioned requested changes. 
Thank you.

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest, Could you please share some updates on the above forementioned requested changes. Thank you.

We are working on to checking it. Will updates you by tomorrow EOD itself.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Thank you for the updates.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
We have locally tested the playbook that you shared with us. We did minor change in the concate part and it gets running successfully. Also there is not hardcoded url related Armttk error. So I have committed the latest playbook templated and package.
Please look into it.

Also we are getting one new KqlValidation error which was not expected. So can you please help us out for that as well?

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Thank you for committing the changes. I'll take a loot at the changes which you committed and at the KQL validation as well. We will provide you some update before 06 September 2023.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
We have committed the yaml file for custom parser and all the validation errors gets resolved.
So can you please proceed further for PR merging as it already took so much time?

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Thank you for the response.
Will examine this PR and update you about the same soonest before 07 September, 2023.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Available for a quick call

**Comment by jayeshprajapaticrest** [comment]
> Hello @jayeshprajapaticrest, Available for a quick call. Its about sample data

Sure, We can connect at 05:30 today.

**Comment by v-prasadboke** [comment]
I'll Schedule a call as per the same

**Comment by jayeshprajapaticrest** [comment]
> I'll Schedule a call as per the same

@v-prasadboke We can connect now if you are available

**Comment by v-prasadboke** [comment]
<div style="margin-top:24px; margin-bottom:20px"><span style="font-size:24px; color:#252424">Microsoft Teams meeting</span>
</div>
<div style="margin-bottom:20px">
<div style="margin-top:0px; margin-bottom:0px; font-weight:bold"><span style="font-size:14px; color:#252424">Join on your computer, mobile app or room device</span>
</div>
<a style="font-size:14px; font-family:'Segoe UI Semibold','Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif; text-decoration:underline; color:#6264a7" class="me-email-headline" href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_MjRjOWQyNTMtYWNjNS00NjMxLTk2OWUtZWYyNWU3N2Y1NDc5%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%2228263f0a-981b-4a56-a4c9-1e91e9ea44fb%22%7d">Click
 here to join the meeting</a> </div>
<div style="margin-bottom:20px; margin-top:20px">
<div style="margin-bottom:4px"><span style="font-size:14px; color:#252424" data-tid="meeting-code">Meeting ID:
<span style="font-size:16px; color:#252424">227 294 715 019</span> </span><br>
<span style="font-size:14px; color:#252424">Passcode: </span><span style="font-size:16px; color:#252424">B5TPd2
</span>
<div style="font-size:14px"><a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://www.microsoft.com/en-us/microsoft-teams/download-app">Download
 Teams</a> | <a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://www.microsoft.com/microsoft-teams/join-a-meeting">
Join on the web</a></div>
</div>
</div>
<div style="margin-bottom:4px">
<div style="margin-bottom:4px"><span style="font-size:14px; color:#252424; font-weight:bold">Or call in (audio only)</span>
</div>
<div style="margin-bottom:4px"><a style="font-size:14px; text-decoration:underline; color:#6264a7" href="tel:+13238494874,,774458782#">[+1 323-849-4874,,774458782#](tel:+13238494874,,774458782#)</a>
<span style="font-size:14px; color:#252424">&nbsp; United States, Los Angeles </span></div>
</div>
<span style="font-size:14px; color:#252424">Phone Conference ID: </span><span style="font-size:16px; color:#252424">774 458 782#
</span>
<div style="margin-bottom:20px"><a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://dialin.teams.microsoft.com/8551f4c1-bea3-441a-8738-69aa517a91c5?id=774458782">Find
 a local number</a> | <a style="font-size:14px; text-decoration:underline; color:#6264a7" class="me-email-link" href="https://dialin.teams.microsoft.com/usp/pstnconferencing">
Reset PIN</a> </div>
<div style="margin-bottom:24px; margin-top:20px"><a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://aka.ms/JoinTeamsMeeting">[Learn More](https://aka.ms/JoinTeamsMeeting)</a>
 | <a style="font-size:14px; text-decoration:underline; color:#6264a7; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif" class="me-email-link" href="https://teams.microsoft.com/meetingOptions/?organizerId=28263f0a-981b-4a56-a4c9-1e91e9ea44fb&amp;tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&amp;threadId=19_meeting_MjRjOWQyNTMtYWNjNS00NjMxLTk2OWUtZWYyNWU3N2Y1NDc5@thread.v2&amp;messageId=0&amp;language=en-US">
Meeting options</a> </div>

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, I've committed some minimal changes.
Please approve and confirm. Also I've added a readme file which have a date modified section please update the date as accordingly.
Thanks.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
I checked the changes and I have few question in it that you have done.
So is it possible to quick connect today if you are available to discuss and getting better understanding on the changes?

**Comment by v-prasadboke** [comment]
Sure @jayeshprajapaticrest, Is 2:00 AM ok.

**Comment by jayeshprajapaticrest** [comment]
Is it fine to connect at 3:00 PM if you are OK with it?

**Comment by v-prasadboke** [comment]
> Is it fine to connect at 3:00 PM if you are OK with it?

Sure, You can join with the same meeting link mentioned above

**Comment by jayeshprajapaticrest** [comment]
> > Is it fine to connect at 3:00 PM if you are OK with it?
> 
> Sure, You can join with the same meeting link mentioned above

Note. Thanks

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
Sorry for the inconvenience  caused.
I have one sudden important meeting scheduled for other task at 03:00 PM. 
So can we reschedule it after 4:00 PM if possible?

**Comment by v-prasadboke** [comment]
Sure we can go for 4:00 PM.

**Comment by v-prasadboke** [comment]
Hello @jayeshprajapaticrest, Waiting for your approval for my committed changes. Thank you.

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke 
I have made one commit with some minor changes in playbook and workbook. Please check and everything is looks good from my end. Please proceed further.

**Review by v-prasadboke — APPROVED** [review]

**Comment by jayeshprajapaticrest** [comment]
@v-prasadboke Thank you for approving the changes. Let's hope finally it will merge by today.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**
